alerting/src/test/kotlin/org/opensearch/alerting/resthandler/SecureDestinationRestApiIT.kt

```diff
@@ -55,29 +64,13 @@ class SecureDestinationRestApiIT : AlertingRestTestCase() {
             email = null
         )
 
-        if (securityEnabled()) {
-            // when security is enabled. No errors, must succeed.
-            val response = client().makeRequest(
-                "POST",
-                "$DESTINATION_BASE_URI?refresh=true",
-                emptyMap(),
-                destination.toHttpEntity()
-            )
-            assertEquals("Create monitor failed", RestStatus.CREATED, response.restStatus())
-        } else {
-            // when security is disable. Must return Forbidden.
-            try {
-                client().makeRequest(
-                    "POST",
-                    "$DESTINATION_BASE_URI?refresh=true",
-                    emptyMap(),
-                    destination.toHttpEntity()
-                )
-                fail("Expected 403 FORBIDDEN response")
-            } catch (e: ResponseException) {
-                assertEquals("Unexpected status", RestStatus.FORBIDDEN, e.response.restStatus())
```

Comment:
how do we plan to validate  ``else block`` of the current test here?

Reply:
These are just made to be run when running with security plugin installed.

Reply:
My question was more around the ``FORBIDDEN response`` validation, which you answered in the other [comment](https://github.com/opensearch-project/alerting/pull/297#discussion_r796179365). Thanks